source/blog/2018-02-06-fedora-atomic-workstation.html.md

```diff
@@ -0,0 +1,172 @@
+---
+title: An Introduction to Fedora Atomic Workstation
+author: jlebon
+date: 2018-02-06 00:00:00 UTC
+published: true
+comments: false
+tags: atomic, fedora, workstation, desktop
+---
+
+For the last few Fedora releases, the Workstation WG has been working on
+combining the best of the Project Atomic pattern with the Fedora Workstation
+Edition into a deliverable dubbed &quot;Fedora Atomic Workstation&quot; (often
+shortened as &quot;Atomic WS&quot;). In Fedora 27, we have reached a point where
+we feel comfortable inviting other developers and enthusiasts to try it out and
+even make it their daily driver.
+
+Read on to discover what Fedora Atomic Workstation is, what its benefits are,
+and how you can get started today! This blog post is based on a talk I gave at
```

Comment:
maybe "Read on to discover the benefits of Fedora Atomic Workstation and how you can get started today!" then new paragraph with

Note: This blog post is based on a talk I gave at DevConf.cz 2018. Head over to [YouTube](https://www.youtube.com/watch?v=7c3GdfhWzcc) if you'd prefer listening.